### PR TITLE
Minor update for developer-workflow.md

### PR DESCRIPTION
**Pull request by girish17**
#### Summary
Added description for `make clean` command

### CODE REVIEW

File: site/content/extend/plugins/developer-workflow.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ subsection: Plugins
 - `make disable` - Disables the plugin on the Mattermost server
 - `make reset` - Disables and re-enables the plugin on the Mattermost server
 - `make debug-plugin` - Starts a `delve` process and attaches it to your running plugin
+- `make clean` - Force deletes recursively the contents of `dist`, `server/coverage.txt`, `server/dist`, `webapp/junit.xml`, `webapp/dist`, `webapp/node_modules` and `build/bin/` folders
```
Comment (hanzei) [inline site/content/extend/plugins/developer-workflow.md:19]: I'm a bit hesitant to name out the directories here as that information can become outdated quite easily. Maybe something like 
```suggestion
- `make clean` - Force deletes the content of build related files. Use when running into build issues,
```

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for updating the docs!

File: site/content/extend/plugins/developer-workflow.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ subsection: Plugins
 - `make disable` - Disables the plugin on the Mattermost server
 - `make reset` - Disables and re-enables the plugin on the Mattermost server
 - `make debug-plugin` - Starts a `delve` process and attaches it to your running plugin
+- `make clean` - Force deletes the content of build related files. Use when running into build issues,
```
Comment (hanzei) [inline site/content/extend/plugins/developer-workflow.md:19]: Typo
```suggestion
- `make clean` - Force deletes the content of build related files. Use when running into build issues.
```

**Review by hanzei — APPROVED** [review]

File: site/content/extend/plugins/developer-workflow.md (hunk 2 above)
Comment (girish17) [reply site/content/extend/plugins/developer-workflow.md:19]: Agree. The folder names could change. It is okay to keep it generic.

Comment (mickmister) [reply site/content/extend/plugins/developer-workflow.md:19]: `build related` should also be `build-related`

Comment (mickmister) [reply site/content/extend/plugins/developer-workflow.md]: Heads up that there's an unresolved request above to change the comma to a period.

**Review by mickmister — APPROVED** [review]
LGTM other than the remaining punctuation request :+1: Thanks for the PR @girish17!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**